pyvista/jupyter/notebook.py

```diff
@@ -108,7 +108,7 @@ def show_ipyvtk(plotter):
 
 
 def show_panel(plotter):
-    """Take the active renderer(s) from a plotter and show them using ``panel``."""
+    """Take the active renderers from a plotter and show them using ``panel``."""
```

Comment:
There's a subtle change in meaning here. Do we have to give this up?

Reply:
Resolved in a separate "conversation". For whatever reason I couldn't add a suggestion to this "conversation."